### DISCUSSION TITLE
could we have `pp` command in pudb?

### DISCUSSION THREAD

**Discussion by dz0** [inducer/pudb#553] [Ideas]
there is watch panel, but I miss the `pp` (as in pdb)

ps.: I know I could do manually from pprint import pprint; pprint(data)`

**Comment by inducer** [comment]
You mean in the interactive shell? Note that what you enter in the interactive shell is just normal Python; it's conceptually not the same as the pdb prompt.

**Comment by asmeurer** [comment]
You could set a custom stringifier to use `pprint`. That could get noisy as the default, but you can press `c` on a variable to switch to the custom stringifier. 

Another option is to use IPython as your shell, which pretty-prints output by default.
